src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -1045,17 +1252,21 @@ public boolean markSupported() {
         }
 
         public void mark(int readLimit) {
```

Comment:
this gives warning, mark is overriding a synchronized method without being synchronized, same with reset